README.md

```diff
@@ -16,77 +16,89 @@ _Files used in current testing development phase:_
 
-```R
-module load shifter
+For the following applications, you'll need to load the singularity and slurm modules:
+
+```bash
+module load singularity slurm
 ```
-Here's how to get the image that Jesse built from Dockerhub and translate it to shifter (this should only need to be done once per image):
-```R
-shifterimg pull docker:jrossusgs/glm3r:v0.6d
+Here's how to get the image that Jesse built from Dockerhub and translate it to Singularity (this has already been done for the image listed below, and should only need to be re-done if a new image is built):
+```bash
+cd /caldera/projects/usgs/water/iidd/datasci/lake-temp/lake-temperature-process-models
+singularity pull docker://jrossusgs/glm3r:v0.7
+# now you can see the singularity image: it is a file called glm3r_v0.7.sif
 ```
-Here's how to build targets using the shifter container and the `targets::tar_make_clustermq(target_name, workers=n_workers)` option to build targets in parallel within the shifter container, with a specified number of workers (up to 80, as Denali has 80 cores per node). The `srun` command will allocate a node and then run the specified command, in this case `Rscript. Targets will then delegate work out to `n_workers` cores for any parallelizable step that you don't specifically tell it to run in serial.
+Here's how to build targets using the Singularity container and the `targets::tar_make_clustermq(target_name, workers=n_workers)` option to build targets in parallel within the Singularity container, with a specified number of workers (up to 72, as Tallgrass has 72 cores per node). The `srun` command will allocate a node and then run the specified command, in this case `Rscript`. Targets will then delegate work out to `n_workers` cores for any parallelizable step that you don't specifically tell it to run in serial.
 ```R
-srun --nodes=1 -t 00:30:00 -A watertemp shifter --image=docker:jrossusgs/glm3r:v0.6d Rscript -e 'targets::tar_make_clustermq(p2_glm_uncalibrated_runs, workers=79)'
+srun --pty -c 10 -A watertemp singularity exec glm3r_v0.7.sif Rscript -e 'targets::tar_make_clustermq(p2_glm_uncalibrated_runs, workers=10)'
 ```
-Here's how to run the shifter container interactively on an allocated job:
+
+Here's how to run the Singularity container interactively on an allocated job:
 ```R
-srun --pty --nodes=1 -t 00:30:00 -A watertemp shifter --image=docker:jrossusgs/glm3r:v0.6d /bin/bash
-# when you're in the shifter environment, the prompt starts with "I have no name!@"
+srun --pty -c 10  -A watertemp singularity exec glm3r_v0.7.sif bash
 R
 library(targets)
 tar_make_clustermq(p2_glm_uncalibrated_runs, workers=79)
 # etc
-q()
-exit
 ```
 
-Alternatively, once a node is allocated it can be ssh'ed to:
+You can also get an interactive RStudio on tallgrass. The best documentation for this is currently [here](https://code.usgs.gov/wma/wp/pump-temperature#running-interactive-sessions-on-hpc). The tl;dr is
+
 ```bash
-jross@denali-login2:~> module load shifter
```

Comment:
I removed this section because (a) `srun` will run `salloc` automatically, so I don't see any real advantage to running the latter by hand, and (b) running it by hand can cause some annoyance if you haven't remembered to load the singularity module beforehand.